lib/wallet/txdb.js

```diff
@@ -1615,7 +1615,8 @@ class TXDB {
 
     for (const {hash, index} of outpoints) {
       const credit = await this.getCredit(hash, index);
-      assert(credit);
+      if (!credit)
```

Comment:
nit: It could be worth adding a comment here that explains the scenario when a coin wouldn't exist, though it should. Due to the multiple queries, and the coin could have been spent in-between.